src/components/security-analysis-result.js

```diff
@@ -134,19 +154,19 @@ const SecurityAnalysisResult = ({ onClickNmKConstraint, result }) => {
                 postContingencyResult.status !== 'CONVERGED'
             ) {
                 rows.push({
-                    contingencyIndex: index,
+                    //contingencyIndex: index,
```

Comment:
Remove this comment if the field is not used anymore

Reply:
Done